core/hybrisadaptor.cpp

```diff
@@ -298,6 +299,15 @@ bool HybrisManager::typeRequiresWakeup(int type)
 
 void HybrisManager::initManager()
 {
+    QString sensorTypes = SensorFrameworkConfig::configuration()->value("hybrisQuirks/doubleStopReader", QString());
+    for (const QString &iter : sensorTypes.split(" ", QString::SkipEmptyParts)) {
```

Comment:
This will break Qt6 build. Could do the same thing we have here around with
#if QT_VERSION >= QT_VERSION_CHECK(5, 15, 0)

Reply:
hmpf. Or, since empty slots are unlikely to produce > 0 int values, might as well skip the SkipEmptyParts